.github/workflows/ci.yml

```diff
@@ -195,9 +195,13 @@ jobs:
 
       - name: Check Template
         run: >-
+          # The linter requires two crates
+          cargo install cargo-dylint dylint-link
           cargo -vV &&
           cargo run --profile debug-ci -- contract --version &&
           cargo run --profile debug-ci -- contract new --target-dir ${{ runner.temp }} foobar &&
           cargo run --profile debug-ci -- contract build --manifest-path=${{ runner.temp }}/foobar/Cargo.toml &&
+          # build with linting, nightly toolchain is used
+          cargo run --profile debug-ci -- contract build --lint --manifest-path=${{ runner.temp }}/foobar/Cargo.toml &&
```

Comment:
It is enough to simply run `build --lint` here. The previous line (L202) could be safely removed.

Reply:
Updated